src/Exception/MissingAliasException.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\ORM\Exception;
+
+use LogicException;
+
+class MissingAliasException extends LogicException implements ORMException
+{
+    public static function create(): self
+    {
+        return new self('Alias is missing on a select part.');
```

Comment:
Would it be possible to get more details here? Like which select part is missing an alias?